cluster/examples/kubernetes/ceph/create-external-cluster-resources.py

```diff
@@ -89,6 +90,15 @@ def gen_arg_parser(cls, args_to_parse=None):
             args_to_parse = sys.argv[1:]
         return argP.parse_args(args_to_parse)
 
+    def _check_conflicting_options(self):
+        if not self._arg_parser.upgrade and not self._arg_parser.rbd_data_pool_name:
+            raise ExecutionFailureException(
+                "Either '--upgrade' or '--rbd-data-pool-name <pool_name>' should be specified")
+        # a user name must be provided while using '--upgrade' option
+        if not self._arg_parser.run_as_user and self._arg_parser.upgrade:
```

Comment:
We also need a check for `self._arg_parser.rgw_pool_prefix` if we cannot find a pool named `default.rgw.meta`.